packages/ketcher-react/src/script/ui/component/view/ToolActions.test.tsx

```diff
@@ -0,0 +1,13 @@
+import toolActions from '../../action/tools';
+
+describe('ToolActions', () => {
+  const toolsWithTitles = Object.values(toolActions).filter(
```

Comment:
Actually, my proposal is to remove `filter`. That's because it doesn't allow to check the `title` attribute. 
If some button doesn't have title, then it will be filtered out and test will pass.